webhooks/virtualmachine/v1alpha1/validation/virtualmachine_validator.go

```diff
@@ -64,6 +64,7 @@ const (
 	invalidNextRestartTimeOnCreate            = "cannot restart VM on create"
 	invalidNextRestartTimeOnUpdate            = "must be formatted as RFC3339Nano"
 	invalidNextRestartTimeOnUpdateNow         = "mutation webhooks are required to restart VM"
+	settingInstanceIDAnnotationNotAllowed     = "adding instance-id annotation is not allowed"
```

Comment:
This error message seems redundant since you use `field.Forbidden` which includes the annotation key path already.